ddev/tests/conftest.py

```diff
@@ -111,8 +111,9 @@ def local_clone(isolation, local_repo) -> Generator[ClonedRepo, None, None]:
     with cloned_repo_path.as_cwd():
         PLATFORM.check_command_output(['git', 'config', 'user.name', 'Foo Bar'])
         PLATFORM.check_command_output(['git', 'config', 'user.email', 'foo@bar.baz'])
+        branch = PLATFORM.check_command_output(['git', 'rev-parse', '--abbrev-ref', 'HEAD']).strip()
```

Comment:
To reviewer, the `strip()` gets rid of the newline character from the output of `git rev-parse --abbrev-ref HEAD`.